test/models/tasks/copy_media_task_test.rb

```diff
@@ -16,14 +16,14 @@
     assert_equal options.keys, %w[callback job_type source destination]
     assert_match(/^sqs:\/\//, options[:callback])
     assert_equal options[:job_type], "audio"
-    assert_equal options[:source], "s3://prx-testing/test/audio.mp3"
+    assert_equal options[:source], task.media_resource.href
```

Comment:
I see that instead of a direct file path, there's a change to using `task.media_resource.href` as a source. I have an idea of why, but could you explain it with some more detail?
(Is `s3` linking a path to the s3 bucket?)

Reply:
Previously, we were alway telling Porter to use the `:source` of the `original_url`.

But I believe in this test, this specific audio (the `task.media_resource` the in the test factory) is in "completed" status.  So instead of the `original_url` (that s3:// file)... this is going to be the `url`.

Seemed better to test for what the `:source` is actually being set to now ... which is whatever `.href` returns.